packages/codemirror-workspace/src/workspace.ts

```diff
@@ -2,46 +2,50 @@ import type { Editor } from "codemirror";
 import { normalizeKeyMap } from "codemirror";
 import {
   CompletionTriggerKind,
-  SignatureHelpTriggerKind,
   DiagnosticTag,
-  TextDocumentSaveReason,
   FileEvent,
+  SignatureHelpTriggerKind,
+  TextDocumentSaveReason,
 } from "vscode-languageserver-protocol";
 
 import { createMessageConnection as createWebSocketMessageConnection } from "@qualified/vscode-jsonrpc-ws";
 import { createMessageConnection as createWorkerMessageConnection } from "@qualified/vscode-jsonrpc-ww";
 import { createLspConnection, LspConnection } from "@qualified/lsp-connection";
 
 import {
-  showDiagnostics,
-  removeDiagnostics,
-  showHoverInfo,
-  removeHoverInfo,
-  hoverInfoEnabled,
   disableHoverInfo,
   enableHoverInfo,
-  showHighlights,
+  gotoLocation,
+  hideCompletions,
+  hoverInfoEnabled,
+  removeDiagnostics,
   removeHighlights,
+  removeHoverInfo,
+  removeSignatureHelp,
+  showDiagnostics,
+  showHighlights,
+  showHoverInfo,
   showInvokedCompletions,
-  showTriggeredCompletions,
-  hideCompletions,
   showSignatureHelp,
-  removeSignatureHelp,
-  gotoLocation,
   showSymbolSelector,
+  showTriggeredCompletions,
 } from "./capabilities";
 import {
   debounce,
+  debouncedBuffer,
+  Disposer,
   filter,
   fromDomEvent,
   map,
   piped,
   skipDuplicates,
-  Disposer,
-  debouncedBuffer,
 } from "./utils/event-stream";
-import { fromEditorEvent, onEditorEvent } from "./events";
-import { lspPosition, lspChange } from "./utils/conversions";
+import {
+  fromEditorEvent,
+  MouseLeaveAllListener,
+  onEditorEvent,
+} from "./events";
+import { lspChange, lspPosition } from "./utils/conversions";
 import { applyEdits } from "./utils/editor";
 import { delay } from "./utils/promise";
 import { removeContextMenu, showContextMenu } from "./ui/context-menu";
```

Comment:
Shoot, looks like my IDE cleaned up the imports here. It's mostly just ordering them. Do you want me to try to revert this section?

Reply:
No, don't worry about it.